src/test/java/org/codehaus/plexus/archiver/jar/module/ModuleDescriptor.java

```diff
@@ -0,0 +1,161 @@
+/**
```

Comment:
I'm definitely not happy with having this class here. Any ideas for better solution? @rfscholte do you think it would be ok to add the means to extract the module version and main class in the Plexus Language? The I can use it as `test` dependency.